compiler/src/dmd/e2ir.d

```diff
@@ -498,6 +499,55 @@ void clearStringTab()
 }
 private __gshared StringTable!(Symbol*) *stringTab;
 
+/*********************************************
+ * Figure out whether a data symbol should be dllimported
+ * Params:
+ *      var = declaration of the symbol
+ * Returns:
+ *      true if symbol should be imported from a DLL
+ */
+bool isDllImported(Dsymbol var)
+{
+    if (target.os & Target.OS.Posix)
+        return false;
```

Comment:
Since dllimport is a windows-only thing, the test should be "if not Windows return false".

Reply:
The same test is used in `createImport()`. I have changed it to Windows-only nonetheless.